ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -146,16 +150,15 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
 
         # Note that the server administrator must add 'www-data' (webapp id) account to 'video' group to access /dev/video*
         # Do not use '777' permission  to avoid a security vulnerability
-        sudo usermod -a -G video www-data
         echo -e "[DEBUG] The group account 'video' includes the below user accounts:"
         cat /etc/group | grep video
 
         # Leave '${REFERENCE_REPOSITORY}/v4l2loopback' directory
         popd    
         
         # Make virtual display on localhost:0.
-        Xvnc :0 &
-        export DISPLAY=0.0:0
+        Xvnc :11 &
+        export DISPLAY=0.0:11
```

Comment:
Please append the below comment to explain why we have to use 11 without 0 (or1). 
```
# To avoid a conflict possibility with existing vnc service ports(5900 ~ 5910), 
# run a vnc service with a port number more than 5911.
```